addons/istio/1.6.x/istio-2.yaml

```
@@ -0,0 +1,67 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: istio
+  labels:
+    kubeaddons.mesosphere.io/name: istio
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.3-2"
```

This revision and the below appversion of istio appear to be disjointed, should this not be:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.4-2"
```
?

---

Done